src/assets/components/Form/Form.tsx

```diff
@@ -30,39 +33,48 @@ export const Form: React.FC<Props> = ({ onSubmit }) => {
   };
 
   return (
-    <div className="form">
-      <form onSubmit={handleSubmit} className="form-content">
-        <h3 className="form-name">Register the Game</h3>
+    <Container
+      sx={{
+        marginLeft: 0,
+        justifyContent: "left",
+        width: 350,
+        border: "1px solid",
+        borderRadius: 3,
+        borderColor: "text.primary",
+      }}
+    >
+      <form onSubmit={handleSubmit}>
```

Comment:
please try to find a way to drop this html tag (i'm not sure what is the best aproach in this case but there is interestion prop called `component` in <Box/> for example, i think it could be simplest solution). But anyway maybe there is some kind of `<Form...` ??